### PR TITLE
Update 2.0 docs to reflect symbol -> prefix change

### PR DESCRIPTION
**Pull request by markcarver**

### CODE REVIEW

File: docs/2.0/extensions/mentions.md [hunk 1]
```diff
@@ -35,25 +35,25 @@ $config = [
         // Sample Input:  `@colinodell`
         // Sample Output: `<a href="https://www.github.com/colinodell">@colinodell</a>`
         'github_handle' => [
-            'symbol'    => '@',
+            'prefix'    => '@',
             'regex'     => '/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}(?!\w)/',
```
Comment (colinodell) [inline docs/2.0/extensions/mentions.md:39]: Good catch!  We also need to remove the starting `/^` and trailing `/` from all the regexes.

Comment (markcarver) [reply docs/2.0/extensions/mentions.md]: Ah, right... done.

Comment (markcarver) [reply docs/2.0/extensions/mentions.md]: This almost makes me wonder if it should be called `pattern` now instead of `regex`?

Comment (colinodell) [reply docs/2.0/extensions/mentions.md]: I like that :)

Comment (markcarver) [reply docs/2.0/extensions/mentions.md]: That can be a follow-up. I don't have time to include that in this PR.

Comment (colinodell) [reply docs/2.0/extensions/mentions.md]: I've created https://github.com/thephpleague/commonmark/issues/569 to track that

**Pull request merged**